timelock.props

```diff
@@ -2,8 +2,6 @@
 # This is supported by Dropwizard starting v1.0.0.
 # http://www.dropwizard.io/1.0.6/docs/manual/configuration.html#http-2-over-tls
 ch.qos.logback:logback-classic = 1.1.7
-com.fasterxml.jackson.*:* = 2.6.7
```

Comment:
I think most of this file should go. Also, lost significant time today since I didn't know this file existed and was confused by the jackson issues :(

Reply:
We ran into all the same problems! I was so confused until I read dependencyInsight more clearly and realized this file existed.

+1 on killing this. The dropwizard-jackson downgrade is also very strange.